modules/oe_editorial_content_lock/oe_editorial_content_lock.install

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Editorial Content Lock module.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_install().
+ */
+function oe_editorial_content_lock_install(): void {
+  // Set default settings for Content Lock.
+  $locked_contents = [
+    'node' => [
+      '*' => '*',
+    ],
+  ];
+  \Drupal::configFactory()
+    ->getEditable('content_lock.settings')
+    ->set('types', $locked_contents)
+    ->save();
```

Comment:
Remember to always add this at the beginning of hook_install() when we update existing configuration within it:
```
  if (\Drupal::isConfigSyncing() === TRUE) {
    // If config is syncing, we do nothing here.
    return;
  }
```
The reason being that , when configuration is syncing (i.e. when running `drush cim`) the configuration object that we are trying to modify (in this case `content_lock.settings`) might not exist yet.